### ISSUE TITLE
Facing issues with the API for using ggmap

### ISSUE THREAD

**Issue by deepmaniyar** [dkahle/ggmap#279]
```
devtools::install_github("dkahle/ggmap", ref = "tidyup", force=TRUE)
devtools::install_github("hadley/ggplot2")
library(ggmap)
library(devtools)
register_google(key = API_KEY)
map <- get_googlemap(center = 'India', zoom = 4)
```

After doing most of the things above I am still getting this error.
My billing is enabled.
I installed the latest ggmap and ggplot2 (removed digest and rlang before that to have the latest version)

_Source : https://maps.googleapis.com/maps/api/staticmap?center=India&zoom=4&size=640x640&scale=2&maptype=terrain&key=xxx
Error in aperm.default(map, c(2, 1, 3)) :
invalid first argument, must be an array
In addition: Warning message:
In get_googlemap(center = "India", zoom = 4) : HTTP 400 Bad Request_

**Comment by dan-reznik** [comment]
yep just had the same problem. looks like get_googlemap doesn't accept a name anymore

**Comment by ahcyip** [comment]
why does ggmap insert "xxx" in front of the API key? and what does "scrub_key" do?

**Comment by ahcyip** [comment]
with a name, you might need the geocoding API as well. (but i'm still getting HTTP 400 bad request)
